bluebrain/repo-bluebrain/packages/py-vascpy/package.py

```diff
@@ -11,6 +11,7 @@ class PyVascpy(PythonPackage):
     datasets """
 
     homepage = "https://github.com/BlueBrain/vascpy"
+    git      = "git@github.com:BlueBrain/vascpy.git"
```

Comment:
this should be still `https` i.e. `https://github.com/BlueBrain/vascpy` (so that package can be cloned with SSH key setup)

Reply:
But most of the other recipes I have seen use the SSH URL. Do you want me to replace it with the HTTPS URL anyway?

Reply:
IIRC, any _official_ Github repo should have HTTPS. For GitLab OTOH, we have SSH for users.

Reply:
_forgot to respond here,_ for public repos, there is no reason to have `ssh` URLs (which will force users to have GitHub account / ssh keys configured).